.env

```diff
@@ -1 +1 @@
-GRPC_SSL_CIPHER_SUITES='HIGH+ECDSA'
+GRPC_SSL_CIPHER_SUITES='HIGH+ECDSA:ECDHE-RSA-AES128-GCM-SHA256'
```

Comment:
I'm not too familiar with this, but is using the `HIGH+ECDSA` for our cert an option?

Reply:
I used [`node-forge`](https://github.com/digitalbazaar/forge) which is purely written in JavaScript for generating the certificate and that library has unfortunately no support for ECDSA signatures

But if we *really* want to use ECDSA we could use a wrapper for the CLI of `openssl` (which is tricky to install on Windows) or write a simple C++ application that generates the certificate with the `openssl` library (shouldn't be a problem to compile since we need `node-gyp` for other dependencies anyways).

What's your opinion? @sangaman

Reply:
Gotcha, doesn't seem like it's worth the trouble. I figure this should be fine, but your idea sounds good if we ever do want to implement it that way.